health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/UserDownload.js

```
@@ -55,20 +55,24 @@ const UserDownload = () => {
 
 
-            <Card>
+            <Card type="secondary" >
                 <HeaderComp title="DOWNLOAD_USER_DATA" styles={{ color: "black" }} />
                 <TextBlock body="DOWNLOAD_DESC" />
                     {data?.ResourceDetails &&
                         [...data?.ResourceDetails].reverse().map((item, index) => {
                             let fileName = item?.additionalDetails?.fileName || `FileNo${item?.processedFilestoreId?.slice(0, 4) || ''}`;
-                            
+                            console.log(typeof String(fileName))
+                            debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)